Add hidden aspects section (#72)
MiniFate/MiniFate: _chapters/05-aspects-and-fate-points.md

#### Hidden Aspects

Most character and story aspects are visible to everyone at the table, but not
all of them. The GM may choose to keep some aspects hidden.  Obvious examples
include monster weaknesses (**Missing Scale Over His Heart**), imminent
ambushes (**Those Bushes Look Suspiciously Like Orcs**), and the schemes and
motivations of NPCs (**Lord Grofhurst Is My Unknowing Puppet**). Hidden
aspects serve more or less as the GM's notes; if invoked or compelled, they
must be revealed to everyone.

Sometimes the story revolves around an NPC with a secret or a hidden
motivation that should be revealed slowly. In this case, it is often best to
start with slightly suspicious public aspects and then change them as the
players catch on.

> > Lord Kellett is a Dragon using his magic to masquerade as the human general
> > of the Empire's armies. He starts with the aspect **Strong as Ten Men**

Reviewer: ```suggestion
> Lord Kellett is secretly a Dragon in human
form. He starts with the aspect **Strong as Ten Men**
```
